src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,172 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    transient int size;
```

Comment:
- why do we use transient here? What does it mean?
- let's add access modifiers